addons/udes_stock/tests/test_limit_orderpoints.py

```diff
@@ -50,11 +52,13 @@ def test02_cannot_add_orderpoint_if_limit_on_parent(self):
         """
         self.op_out_location.u_limit_orderpoints = True
         with self.assertRaises(ValidationError) as cm:
-            self.create_orderpoint(self.banana, self.op_test_output_location_01, 5, 10)
+            self.create_orderpoint(
+                self.banana, self.op_test_output_location_01, 5, 10)
         ex = cm.exception
         self.assertEqual(
             "An order point for location {} already exists on "
-            "{}.".format(self.op_test_output_location_01.name, self.apple.name),
+            "{}.".format(self.op_test_output_location_01.name,
+                         self.apple.name),
             ex.name,
```

Comment:
Shouldnt be change here as well deprecated name

Reply:
Good spot - missed that in the logs